docs/rules/keyword-spacing.md

```diff
@@ -19,59 +19,92 @@ Of course, you could also have a style guide that disallows spaces around keywor
 
 ## Rule Details
 
-This rule will enforce consistency of spacing around keywords and keyword-like tokens: `as` (in module declarations), `break`, `case`, `catch`, `class`, `const`, `continue`, `debugger`, `default`, `delete`, `do`, `else`, `export`, `extends`, `finally`, `for`, `from` (in module declarations), `function`, `get` (of getters), `if`, `import`, `in`, `instanceof`, `let`, `new`, `of` (in for-of statements), `return`, `set` (of setters), `static`, `super`, `switch`, `this`, `throw`, `try`, `typeof`, `var`, `void`, `while`, `with`, and `yield`.
+This rule enforces consistent spacing around keywords and keyword-like tokens: `as` (in module declarations), `break`, `case`, `catch`, `class`, `const`, `continue`, `debugger`, `default`, `delete`, `do`, `else`, `export`, `extends`, `finally`, `for`, `from` (in module declarations), `function`, `get` (of getters), `if`, `import`, `in`, `instanceof`, `let`, `new`, `of` (in for-of statements), `return`, `set` (of setters), `static`, `super`, `switch`, `this`, `throw`, `try`, `typeof`, `var`, `void`, `while`, `with`, and `yield`.
 
-The following patterns are considered problems:
+
+## Options
+
+This rule has an object option:
+
+* `"before"` (default `true`) enforces consistent spacing style before keywords
```

Comment:
Good point. What if we standardized them so that:

 * if boolean value is true by default
    * include `"option": true` (default) description
    * include `"option": false` description
 * if boolean is false by default
    * include `"option": true` description

I can add this to the guidelines if you agree (there might be cases I haven't thought of that may need both in the latter case or some other exception)